src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusAlgorithmLevitsky.cpp

```diff
@@ -114,11 +114,10 @@ static void registerHit(int* data, char c) {
 }
 
 MSAConsensusAlgorithmLevitsky::MSAConsensusAlgorithmLevitsky(MSAConsensusAlgorithmFactoryLevitsky* f, const MultipleAlignment& ma, bool ignoreTrailingLeadingGaps, QObject* p)
-    : MSAConsensusAlgorithm(f, ignoreTrailingLeadingGaps, p) {
-    globalFreqs.resize(256);
-    memset(globalFreqs.data(), 0, globalFreqs.size() * 4);
-
-    int* freqsData = globalFreqs.data();
+    : MSAConsensusAlgorithm(f, ignoreTrailingLeadingGaps, p)
+    , globalFreqs(QVarLengthArray<int>(256)) {
+    int *freqsData = globalFreqs.data();
+    std::fill(freqsData, freqsData + 256, 0);
```

Comment:
You do not need a raw pointer and magic constants.
Use the classic code I pointed in the previous message: std:fill(globalFreqs.begin(), globalFreqs.end(), 0).
Please, update all places like this in your patch.